repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py

```diff
@@ -8,7 +8,8 @@
 
 SAP_HANA_MINIMAL_MAJOR_VERSION = 2
 SAP_HANA_RHEL8_REQUIRED_PATCH_LEVELS = ((5, 54, 0),)
-SAP_HANA_MINIMAL_VERSION_STRING = 'HANA 2.0 SPS05 rev 54 or later'
+SAP_HANA_RHEL9_REQUIRED_PATCH_LEVELS = ((5, 59, 4), (6, 63, 0))
+SAP_HANA_MINIMAL_VERSION_STRING = 'HANA 2.0 SPS05 rev 59.04 or later, or SPS06 rev 63 or later'
```

Comment:
This is a 8to9 minimal_version_string. For 7to8 it should stay original: 'HANA 2.0 SPS05 rev 54 or later'